configs/application/components.json

```diff
@@ -14,26 +14,32 @@
 				"height": 432
 			},
 			"component": {
-				"inject": false,
+				"displayName": "Welcome Component",
 				"spawnOnStartup": false,
+				"inject": false,
```

Comment:
Is inject false needed? I feel like we should remove it since inject isn't the preferred mechanism.

Reply:
Agreed, removed.